code/modules/mob/living/simple_animal/slime/life.dm

```diff
@@ -8,7 +8,7 @@
 
 
 /mob/living/simple_animal/slime/Life()
-	set invisibility = 0
+	set invisibility = MOB_INVIS_DEFAULT
```

Comment:
I suppose, but I mean the line had a reason to exist in the first place though right? And if the system gets more use there might be a reason to go back and say "ok maybe this mob type should be changed"